### DISCUSSION TITLE
Seedvault code reissue

### DISCUSSION THREAD

**Discussion by thisIsMeHello** [seedvault-app/seedvault#190] [Q&A] [answered]
Hi, thanks for this utility, I haven't needed to use the restoration function yet but assuming it works, this is such a useful addition!

I've lost my 12 word code. How do I initiate a new backup with a new code?

Thanks,
Marcus

**Comment by grote** [comment]
This will be possible once https://github.com/seedvault-app/seedvault/pull/168 is merged.

**Reply by grote** [reply]
For now, you can find Seedvault in System Settings -> App and clear all its data, so you can start fresh with a new code.

**Comment by dznsm** [comment, marked as answer]
Something like -
android settings > apps > system apps > seedvault > clear app data 
Delete any backup you have.
Start up seedvault and write down your new 12 words.
(Edit: grote just beat me to it)

**Reply by thisIsMeHello** [reply]
Thanks for the reply
(for the similarly challenged)
settings > apps and notifications > see all apps > (three dots on right-up corner and select) Show System > seedvault > storage & cache - then clear storage
